src/snowcli/cli/core/stage.py

```diff
@@ -49,7 +49,7 @@ def stage_get(
     environment: str = EnvironmentOption,
     name: str = typer.Argument(..., help="Stage name"),
     path: Path = typer.Argument(
-        Path("."),
+        Path(".."),
```

Comment:
Why? This should download the file to current working directory of user

Reply:
Maybe using `Path.cwd()` will be more explicite?

Reply:
Idea changed it 😮 , I didn't even notice it. Changed to `Path.cwd()`